lib/app/admin/data-demo/file.js

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Might be good to add comments to these files to explain that these are sample data sets. 

Developers may or may not want to use there.